### PR TITLE
Fix/cleanup files

### PR DESCRIPTION
**Pull request by IsaacGemal**
Added some minor tweaks and clarifications to the readme, and a complete, simple house.js file as an example to make things even easier to get set up and running.

### CODE REVIEW

**Review by huntcsg — APPROVED** [review]
ship it!

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 builds
 data
+*.lockb
```
Comment (jannalulu) [inline .gitignore:4]: if we are using bun, we should have the same lock file to control for dependency versions, no?

File: README.md [hunk 2]
```diff
@@ -7,6 +7,21 @@ This is a simple demo repo that includes:
...
 
+# Example Build
+
+The `builds/house.js` provides an example of a complete building script that creates a simple house structure. You can use this as a reference for how the building functions work and what a completed script looks like.
+
+# Installation
+
+Before getting started, install the required dependencies:
+
+```shell
```
Comment (jannalulu) [inline README.md:18]: shell seems redundant here

File: .gitignore (hunk 1 above)
Comment (IsaacGemal) [reply .gitignore:4]: That's a good point. Do we want to use bun or npm going forward?
Personally I use them interchangeably in my projects more often than not (which probably isn't ideal)

Comment (jannalulu) [reply .gitignore:4]: I'd prefer bun because it has better Window-Mac interchangeability in my experience, but I don't feel super strongly about this

**Pull request merged**